src/Repository/Repository.php

```diff
@@ -80,8 +58,13 @@ public function getNeedsReviewUrl()
         );
     }
 
-    public function getFullName()
+    public function getFullName(): string
     {
         return sprintf('%s/%s', $this->getVendor(), $this->getName());
     }
+
+    public function __toString(): string
+    {
+        return $this->getFullName();
+    }
```

Comment:
If we can, I'd rather not have to rely on a `__toString` - i.e. let's explicitly call `getFullName()` wherever we're using this.